templates/IAM/service-account.yaml

```diff
@@ -1,20 +1,62 @@
 Description: Setup an AWS service account and role
 AWSTemplateFormatVersion: 2010-09-09
 Parameters:
-  ServiceUserPolicies:
```

Comment:
Renaming the parameter is gonna break back-compat, problem?

Reply:
i pin it to a tag that has the old version.  https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/88